synth.py

```diff
@@ -34,4 +34,7 @@
       destination_name='bigquerystorage',
   )
 
-java.common_templates()
+java.common_templates(excludes=[
+    #TODO: REMOVE THIS WHEN GRPC FIXES ITS TEMPLATE
+    'google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1alpha2/MockBigQueryWriteImpl.java'
```

Comment:
This file is not a common template, so this won't work. The client copying happens in the `java.bazel_library` command above and does not include an option for excluding files.

To fix in synth, you will want to use `s.replace(files, search_regex, replace_string)`

Reply:
Done.